Framework/src/CheckRunner.cxx

```diff
@@ -254,10 +250,12 @@ void CheckRunner::run(framework::ProcessingContext& ctx)
   updateRevision();
 
   // monitoring
-  endLastObject = system_clock::now();
   if (timer.isTimeout()) {
     timer.reset(1000000); // 10 s.
-    mCollector->send({ mTotalNumberHistosReceived, "objects" }, o2::monitoring::DerivedMetricMode::RATE);
+    mCollector->send({ mTotalNumberHistosReceived, "qc_objects_received" }, DerivedMetricMode::RATE);
+    mCollector->send({ mTotalNumberCheckExecuted, "qc_checks_executed" }, DerivedMetricMode::RATE);
+    mCollector->send({ mTotalNumberQOStored, "qc_qo_stored" }, DerivedMetricMode::RATE);
+    mCollector->send({ mTotalNumberMOStored, "qc_mo_stored" }, DerivedMetricMode::RATE);
```

Comment:
Perhaps we should stick either to `objects` or `mo`, since we mean the same. In TaskRunner we use `objects`.

Reply:
actually I had a commit that I forgot to push that did exactly that.

Reply:
wait, I meant another one, here I wanted to differentiate MOs from QO stored. I could merge them and then I would call them objects.

Reply:
Differentiating MOs and QOs makes sense completely. I was more reluctant to use `objects` and `mo` interchangeably. However, as I think of it now, we will allow not only MOs here, but also other kinds of objects might be injected here, which won't be anyway stored. So the distinction is actually justified here.